### PR TITLE
43 email list

### PR DESCRIPTION
**Pull request by smebrahimi**
This address issues
 [#43](https://github.com/whynotearth/Volkswagen-email-corperate-communications/issues/39)
 [#44](https://github.com/whynotearth/Volkswagen-email-corperate-communications/issues/44) 
[#45](https://github.com/whynotearth/Volkswagen-email-corperate-communications/issues/45)
[#46](https://github.com/whynotearth/Volkswagen-email-corperate-communications/issues/46)
[#42](https://github.com/whynotearth/Volkswagen-email-corperate-communications/issues/42) 

https://github.com/whynotearth/Volkswagen-email-corperate-communications/issues/43

### CODE REVIEW

File: src/views/EmailCampaign.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import BaseAppBarHeader from '@/components/BaseAppBarHeader.vue';
+
+export default {
+  name: 'EmailCampaign',
+  components: { BaseAppBarHeader }
+};
+</script>
+
+<style scoped>
+.settingsOptions {
```
Comment (mort3za) [inline src/views/EmailCampaign.vue:62]: @smebrahimi Please use text styles in typography.css

**Comment by mort3za** [comment]
image

Make the link to fill height of the list items. Currently only the text is clickable.

File: src/views/EmailListItem.vue [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="">
+    <BaseAppBarHeader title="Email" :to-link="backLink">
+      <div class="flex-grow inline-block text-right">
+        <img
```
Comment (mort3za) [inline src/views/EmailListItem.vue:5]: @smebrahimi Use hyperlink for buttons and clickable things please. also needs `cursor-pointer` class.

File: src/views/Settings.vue [hunk 3]
```diff
@@ -75,4 +112,7 @@ export default {
   font-size: 16px;
   line-height: 20px;
 }
+.bb-1 {
```
Comment (mort3za) [inline src/views/Settings.vue:115]: `border-b-1` in tailwind

File: src/views/Settings.vue [hunk 4]
```diff
@@ -34,7 +70,7 @@
             fill-rule="evenodd"
             clip-rule="evenodd"
             d="M7.4 1.4L6 0L0 6L6 12L7.4 10.6L2.8 6L7.4 1.4Z"
-            fill="currentColor"
```
Comment (mort3za) [inline src/views/Settings.vue:37]: @smebrahimi Please don't change currentColor, instead use a text color class on svg. (currentColor === text color)

File: src/views/Settings.vue (hunk 3 above)
Comment (smebrahimi) [reply src/views/Settings.vue:115]: have a .border-b-2 not .border-b-1

Comment (mort3za) [reply src/views/Settings.vue:115]: Yeah, make sure to add border color too

**Pull request merged**